.circleci/config.yml

```
@@ -1,91 +1,72 @@
 version: 2.1
 orbs:
   pack: buildpacks/pack@0.2.4
-  ruby: circleci/ruby@1.1.2
-  shellcheck: circleci/shellcheck@2.2.0
 
-references:
-  hatchet: &hatchet
-    run:
-      name: Run test suite
-      command: PARALLEL_SPLIT_TEST_PROCESSES=25 IS_RUNNING_ON_CI=1 bundle exec parallel_split_test spec/hatchet
-  pack_cnb: &pack_cnb
-    run:
-      name: Run test suite
-      command: bundle exec rspec spec/cnb
-  docker_commands: &docker_commands
-    run:
-      name: Run test suite
-      command: PARALLEL_SPLIT_TEST_PROCESSES="4" IS_RUNNING_ON_CI=1 bundle exec parallel_split_test spec/docker
-  unit: &unit
-    run:
-      name: Run test suite
-      command: PARALLEL_SPLIT_TEST_PROCESSES="4" IS_RUNNING_ON_CI=1 bundle exec parallel_split_test spec/unit
-  hatchet_setup: &hatchet_setup
-    run:
-      name: Hatchet setup
-      command: |
-        bundle exec hatchet ci:setup
+executors:
+  rust:
+    machine:
+      image: ubuntu-2004:202201-02
```

There's an Ubuntu 22.04 image out now - can also use the `current` tag to save having to update this all the time :-)

```suggestion
      image: ubuntu-2204:current
```

(https://circleci.com/developer/machine/image/ubuntu-2204)